crates/astria-conductor/src/data_availability/sync.rs

```diff
@@ -0,0 +1,191 @@
+use std::time::Duration;
+
+use celestia_client::{
+    celestia_types::{
+        nmt::Namespace,
+        Height,
+    },
+    jsonrpsee::http_client::HttpClient,
+    CelestiaClientExt as _,
+    CelestiaSequencerBlob,
+};
+use color_eyre::eyre::{
+    self,
+    Report,
+    WrapErr as _,
+};
+use futures::stream::FuturesOrdered;
+use tokio::select;
+use tracing::{
+    error,
+    info,
+    instrument,
+    warn,
+};
+use tryhard::RetryFutureConfig;
+
+use super::block_verifier::BlockVerifier;
+use crate::{
+    data_availability::{
+        send_sequencer_subsets,
+        verify_sequencer_blobs_and_assemble_rollups,
+    },
+    executor,
+};
+
+pub(crate) async fn find_da_block_with_sequencer_data(
```

Comment:
What does this function do? Loop from `start_height..start_height+da_block_range` until it finds the first sequencer namespace?

I think it should probably be called `find_height_with_sequencer_blob`